packages/server-wallet/src/fp.ts

```diff
@@ -0,0 +1,2 @@
+export {Either, left, right} from 'fp-ts/lib/Either';
```

Comment:
My vote is to avoid this indirection and import directly from `fp-ts` in the module that needs the import.

Reply:
Resolved via da96814f9